ivy/functional/backends/torch/experimental/creation.py

```diff
@@ -245,3 +245,33 @@ def hz_to_mel(f):
     upper_slopes = (upper_edge_mel - spec_bin_mels) / (upper_edge_mel - center_mel)
     mel_weights = torch.maximum(zero, torch.minimum(lower_slopes, upper_slopes))
     return torch.nn.functional.pad(mel_weights, (0, 0, 1, 0))
+
+
+def unsorted_segment_mean(
+    data: torch.Tensor,
+    segment_ids: torch.Tensor,
+    num_segments: Union[int, torch.Tensor],
+) -> torch.Tensor:
+    # Check if the parameters are valid
+    ivy.utils.assertions.check_unsorted_segment_min_valid_params(
```

Comment:
could you please rename the original `check_unsorted_segment_min_valid_params` fn to `check_unsorted_segment_valid_params` and use `check_unsorted_segment_valid_params` everywhere instead?

Reply:
done

Reply:
ah sorry it's not clear - you'll have to rename all instances of `check_unsorted_segment_min_valid_params` in ivy codebase to `check_unsorted_segment_valid_params`